react/CozyTheme/CozyThemeProvider.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useContext } from 'react'
+import merge from 'lodash/merge'
+import { ThemeProvider as MuiThemeProvider } from '@material-ui/core/styles';
+
+import CozyThemeContext from './CozyThemeContext'
+import stylesFromTheme from './styleFromTheme'
+import muiThemeFromCozyTheme from './muiThemeFromCozyTheme'
+
+export default function CozyThemeProvider({ theme, children }) {
+  const context = useContext(CozyThemeContext)
+  const newTheme = merge({}, theme, context)
+  const styles = stylesFromTheme(context.initialized ? theme : newTheme)
+  const muiTheme = muiThemeFromCozyTheme(theme, context, newTheme)
+  newTheme.initialized = true
```

Comment:
I don't understand this `initialized` attribute ?

Reply:
it's related to line 12, in order to initialize a full cozy theme with the defaults at first use (but only at first use)